documentation/content/content-nodes.html

```diff
@@ -213,94 +217,69 @@ <h3 id="master-election">Master election</h3>
-For instance it may know that of the currently existing bucket copies,
-a given copy has been available all the time and may be trusted to have all the information,
-while another copy may be on a storage node that recently restarted,
-so that copy may lack some documents.
-Such historic state is not persisted, and is thus lost on distributor restarts.
 </p><p>
-The distributors use the bucket metadata to ensure the buckets are in a good state.
+</p>
+
+
+<h3 id="maintenance-operations">Maintenance operations</h3>
+<p>
+Distributors track which content nodes have which buckets in their bucket batabase.
+Distributors then use the <a href="idealstate.html">ideal state algorithm</a>
+to generate bucket <em>maintenance operations</em>.
+A stable system has all buckets located per the ideal state:
 <ul>
-  <li>If buckets have too few copies, new copies are generated</li>
-  <li>If buckets have too many copies, superfluous copies are deleted once the
-    distributor knows the copies to delete don't contain data other copies do not have</li>
-  <li>If all the copies do not contain the same data,
-    merges are issued to get bucket copies consistent</li>
+    <li>If buckets have too few replicas, new are generated on other content nodes.</li>
+  <li>If the replicas differ, a bucket merge is issued to get replicas consistent.
+      The distributor sets a bucket replica <em>trusted</em>. <!-- ToDo: need something here - this is confusing - old text below:
+      
+      For instance it may know that of the currently existing replicas,
+      a given replica has been available all the time and may be trusted to have all the information,
+      while another replica may be on a content node that recently restarted,
+      so that replicas may lack some documents.
+      Such historic state is not persisted,
+      and is thus lost on distributor restarts.--></li>
+  <li>If a buckets has too many replicas, superfluous are deleted.
+      Buckets are merged, if inconsistent, before deletion.</li>
   <li>If two buckets exist, such that both may contain the same document,
-    the buckets are split or joined to remove such overlapping buckets</li>
-  <li>If buckets contain too little or too much data, they should be joined or split</li>
-  <li>If not exactly one copy is marked active, and the backend wants a copy to
-    be marked active, activate or deactivate copies to get in a good state</li>
+      the buckets are split or joined to remove such overlapping buckets.
+      Read more on <a href="buckets.html">inconsistent buckets</a>.</li> <!-- ToDo: change this to a proper anchor within buckets.html - this needs a dedicated section, relevant to point to from visiting -->
+  <li>If buckets are too small/large, they will be joined or split.</li>
+  <li>If not exactly one replica is marked active, <!-- ToDo: what is this ?-->
+      and the content node wants a replica to be marked active,
+      activate or deactivate replicas to get in a good state.</li>
```

Comment:
In clusters a with document type marked as `mode="index"` the distributor will ensure that only one of a bucket's replicas within a group will be used when processing search queries. Otherwise search results would contain many duplicates. This is known as an "active" replica, the rest being "inactive" and available for failover if the currently active node should become unavailable, or the ideal state changes.

Reply:
of course - it is that _active_ - will link to the proton doc for details here. thanks!